src/main/java/org/frc5687/rapidreact/commands/catapult/IdleCatapult.java

```diff
@@ -24,7 +24,6 @@ public void initialize() {
     public void execute() {
         super.execute();
         _catapult.setWinchMotorSpeed(0);
-        _catapult.setSpringMotorSpeed(_oi.getSpringMotorSpeed());
     }
```

Comment:
Sorry to nit-pick, but shouldn't this also set speed to 0 like you did for setWinchMotorSpeed?

Reply:
It shouldn't matter too much as motion magic is treated differently, ALSO IdleCatapult is never called.